src/components/Psychologist/Dates/Dates.jsx

```diff
@@ -72,7 +85,41 @@ function Dates() {
 								<p>No tienes citas pendientes</p>
 							</div>
 						)}
-
+				</div>
+			</div>
+			<div className={styles.calendar}>
+				<Calendar
+					onChange={setTgl}
+					tileClassName={({ date }) => {
+						let day = date.getDate();
+						let month = date.getMonth() + 1;
+						if (date.getMonth() < 10) {
+							month = `0${month}`;
+						}
+						if (date.getDate() < 10) {
+							day = `0${day}`;
+						}
+						const realDate = `${date.getFullYear()}/${month}/${day}`;
+						if (dataD.find((val) => val === realDate)) {
+							return styles.highlight;
+						}
+						return 0;
+					}}
```

Comment:
in order to improve readability, please create a function and use it here instead of create the anonymous function here

Reply:
.